src/main/java/org/tron/core/db/BandwidthProcessor.java

```diff
@@ -219,9 +219,12 @@ private boolean useAssetAccountNet(Contract contract, AccountCapsule accountCaps
     } catch (Exception ex) {
       throw new RuntimeException(ex.getMessage());
     }
-    String tokenID = ByteArray.toStr(assetName.toByteArray());
+
     AssetIssueCapsule assetIssueCapsule;
     assetIssueCapsule = dbManager.getAssetIssueStoreFinal().get(assetName.toByteArray());
+    String tokenName = ByteArray.toStr(assetName.toByteArray());
+    String tokenID = assetIssueCapsule.getId();
+
     if (assetIssueCapsule == null) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false" ([squid:S2583](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2583))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1746&issues=AWcqs5ppd6oxh3R419JT&open=AWcqs5ppd6oxh3R419JT)

[ISSUE_KEY]: AWcqs5ppd6oxh3R419JT